client/components/post-stats-card/style.scss

```diff
@@ -3,7 +3,7 @@
 @import "@automattic/typography/styles/fonts";
 @import "@automattic/components/src/styles/mixins";
 @import "@automattic/components/src/styles/typography";
-@import "@automattic/components/src/highlight-cards/variables";
+@import "calypso/my-sites/stats/components/highlight-cards/variables";
```

Comment:
Not ideal, but better than imporrting from a separate package 😅

Reply:
It's kinda strange how these variables are defined as being specific to HighlightCards, but are pretty general-purpose and used for a lot of different stats styles ("vertical margin", "mobile breakpoint", etc.). Not necessarily in this pull request, but maybe they'd be better off somewhere more central to stats styling.

Reply:
I agree, and it's quite terrible. I won't have a chance to fix this, but it's a good one to follow up on for anyone who has some spare cycles.